core/src/components/slider/slider.spec.ts

```diff
@@ -66,25 +77,25 @@ describe(`Slider basic`, () => {
 		const sliderDirective = slider.directives.sliderDirective(sliderElement);
 		const minLabelDirective = slider.directives.minLabelDirective(minLabel);
 		const maxLabelDirective = slider.directives.maxLabelDirective(maxLabel);
-		const unsubscribe = slider.state$.subscribe((newState) => {
-			state = newState;
+
+		normalizedState$ = computed(() => {
+			return normalizeState(slider.state$());
 		});
 
 		return () => {
 			sliderDirective!.destroy?.();
 			minLabelDirective!.destroy?.();
 			maxLabelDirective!.destroy?.();
-			unsubscribe();
 		};
 	});
 
 	test(`should create the default configuration for the model`, () => {
-		expect(state).toStrictEqual(defaultStateValues);
+		expect(normalizedState$()).toStrictEqual(defaultStateValues());
 	});
 
 	test(`should recompute clean value if min has changed`, () => {
-		const expectedStateValue = {...defaultStateValues};
-		expect(state).toStrictEqual(expectedStateValue);
+		const expectedStateValue = defaultStateValues();
```

Comment:
Not directly linked to this PR, but can you change all the `StateValue` expressions (expectedStateValue, defaultStateValues) to `State` (expectedState, defaultState). These two expressions has almost the same meaning.